content/plugins/coolsam-signature-pad.md

```diff
@@ -0,0 +1,14 @@
+---
+name: Signature Pad
+slug: coolsam-signature-pad
+author_slug: coolsam
+categories: [form-builder, form-field]
+description: A beautiful Signature Pad Field for Filamentphp Forms
+discord_url: https://discord.com/channels/883083792112300104/1059009209951342632
+docs_url: https://raw.githubusercontent.com/savannabits/filament-signature-pad/3.x/README.md
+github_repository: savannabits/filament-signature-pad
+has_dark_theme: true
+has_translations: true
+image: coolsam-signature-pad.png
+versions: [2, 3]
+---
```

Comment:
Missing `publish_date` for the date that this plugin was first released

Reply:
Got it, Amending.